src/sentry/api/endpoints/event_file_committers.py

```diff
@@ -0,0 +1,168 @@
+                        version=version,
+                    ),
+                )
+            )
+        except Release.DoesNotExist:
+            return None
+
+        return list(commits)
+
+    def _get_commit_file_changes(self, commits, path_name_set):
+        # build a single query to get all of the commit file that might match the first n frames
+
+        pathQuery = reduce(operator.or_, (
+            Q(filename__endswith=next(tokenize_path(path)))
+            for path in path_name_set
+        ))
+
+        query = Q(commit__in=commits) & pathQuery
+
+        commit_file_change_matches = CommitFileChange.objects.filter(query)
+
+        return list(commit_file_change_matches)
+
+    def _match_commits_frame(self, commit_file_changes, frame):
+        #  find commits that match the run time path the best.
+
+        matching_commits = {}
+        best_score = 0
+        for file_change in commit_file_changes:
+            score = score_path_match_length(file_change.filename, frame['abs_path'])
+            if score > best_score:
+                # reset matches for better match.
+                best_score = score
+                matching_commits = {}
+            if score == best_score:
+                #  we want a list of unique commits that tie for longest match
+                matching_commits[file_change.commit.id] = file_change.commit
+
+        return matching_commits.values()
+
+    def _get_committers(self, annotated_frames, commits):
+        # extract the unique committers and return their serialized sentry accounts
+        committers = defaultdict(int)
+
+        limit = 5
+        for annotated_frame in annotated_frames:
+            if limit == 0:
+                break
+            for commit in annotated_frame['commits']:
+                committers[commit.author.id] += limit
+                limit -= 1
+                if limit == 0:
+                    break
+
+        # organize them by this heuristic (first frame is worth 5 points, second is worth 4, etc.)
+        sorted_committers = sorted(committers, key=committers.get)
+        sentry_user_dict = get_users_for_commits(commits)
+
+        return [sentry_user_dict[author_id] for author_id in sorted_committers]
+
+    def get(self, _, project, event_id):
+        """
+        Retrieve Committer information for an event
+        ```````````````````````````````
+
+        Return commiters on an individual event, plus a per-frame breakdown.
+
+        :pparam string project_slug: the slug of the project the event
+                                     belongs to.
+        :pparam string event_id: the hexadecimal ID of the event to
+                                 retrieve (as reported by the raven client).
+        :auth: required
+        """
+        try:
+            event = Event.objects.get(
+                id=event_id,
+                project_id=project.id,
+            )
+            Event.objects.bind_nodes([event], 'data')
```

Comment:
Pull this out of the `try...except` since this woul dbe a different exception anyways.